Data/symbol-properties/symbol-properties-database.csv

```diff
@@ -147,14 +152,35 @@ cme,IBV,future,USD-Denominated Ibovespa Index Futures,USD,1.0,5.0,1.0
 cme,J7,future,E-mini Japanese Yen Futures,USD,6250000.0,1e-06,1.0
 cme,LBS,future,Random Length Lumber Futures,USD,110.0,0.1,1.0
 cme,LE,future,Live Cattle Futures,USD,40000.0,0.025,1.0,,1,100
+cme,M2K,future,Micro E-mini Russell 2000 Index Futures,USD,5.0,0.1,1.0,,1.0,100.0
+cme,M6A,future,Micro Australian Dollar/U.S. Dollar (AUD/USD) Futures,USD,10000.0,0.0001,1.0,,1.0,100.0
+cme,M6B,future,Micro British Pound Sterling/U.S. Dollar (GBP/USD) Futures,USD,6250.0,0.0001,1.0,,1.0,100.0
+cme,M6C,future,Micro USD/CAD Futures,CAD,10000.0,0.0001,1.0
+cme,M6E,future,Micro Euro/U.S. Dollar (EUR/USD) Futures,USD,12500.0,0.0001,1.0,,1.0,100.0
+cme,M6J,future,Micro USD/JPY Futures,JPY,10000.0,0.01,1.0
+cme,M6S,future,Micro USD/CHF Futures,CHF,10000.0,0.0001,1.0
+cme,MBT,future,Micro Bitcoin Futures,USD,0.1,5.0,1.0,,1.0,100.0
```

Comment:
Bitcoin needs the 100x `price_magnifier`? Same question for `MES` since `ES` doesn't?